client/my-sites/upgrades/domain-management/dns/dns-record.jsx

```diff
@@ -1,7 +1,9 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React from 'react';
+import classNames from 'classnames';
+import endsWith from 'lodash/string/endsWith';
```

Comment:
Actually, yes, I believe we should use lodash here. There's been problems with `endsWith` [in the past](https://github.com/Automattic/wp-calypso/pull/1777) (in related news, `startsWith` recently caused [quite a ruckus](https://calypsop2.wordpress.com/2016/01/13/calypso-was-broken-in-ie11safari8-since-yesterday/) too) because it's not polyfilled (last time I checked), so it will break on older browsers.